themes-default/slim/views/editShow.mako

```diff
@@ -133,40 +113,40 @@ const startVue = () => {
                     }
                 };
             },
-            onChangeIgnoredWords: function(items) {
+            onChangeIgnoredWords(items) {
 		        console.debug('Event from child component emitted', items);
```

Comment:
You can remove all the `console.debug` statements.

@p0psicles if you need to debug events use the Vue Devtools Chrome extension.